src/dash/DashMetrics.js

```diff
@@ -33,6 +33,7 @@ import ManifestModel from '../streaming/models/ManifestModel';
 import DashManifestModel from './models/DashManifestModel';
 import FactoryMaker from '../core/FactoryMaker';
 import * as MetricsList from './constants/DashMetricsList';
+const round10 = require('round10').round10;
```

Comment:
> I have used a module from npm which we have done with isoboxer but not something we do all the time in this project, and I'm not really sure why. Perhaps the project grandees have an opinion.

Maybe I should read first -  This is the first require used in the project,  I am very open just want to stay consistent.   my two bits...